go/teams/merkle_test.go

```diff
@@ -83,19 +82,14 @@ func TestMerkleWithHidden(t *testing.T) {
 	require.Equal(t, team.ID, leaf.TeamID, "team id mismatch")
 	require.Equal(t, team.chain().GetLatestSeqno(), leaf.Private.Seqno)
 	require.Equal(t, team.chain().GetLatestLinkID(), leaf.Private.LinkID.Export())
-	require.True(t, hiddenResp.RespType == libkb.MerkleHiddenResponseTypeABSENCEPROOF || hiddenResp.RespType == libkb.MerkleHiddenResponseTypeOK)
+	require.True(t, hiddenResp.RespType == libkb.MerkleHiddenResponseTypeOK)
 	require.EqualValues(t, 1, hiddenResp.UncommittedSeqno)
-	if hiddenResp.RespType == libkb.MerkleHiddenResponseTypeABSENCEPROOF {
+	if hiddenResp.RespType == libkb.MerkleHiddenResponseTypeOK {
```

Comment:
You can remove the if and the whole else clause. resptype is now guaranteeed to be OK